src/templates/_airportGuide.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="airportGuide-container nice-scrollbar">
```

Comment:
@sam-irl (part 2).  If we can have the markup in the DOM, we should put it in the DOM.  In other words, if we don't need to generate markup via js, then we probably shouldn't.